BUILD.md

```diff
@@ -161,32 +161,24 @@ mamba activate raft_env_name
 The Python APIs can be built using the `build.sh` script:
 
 ```bash
-./build.sh raft-dask pylibraft
-```
-
-`setup.py` can also be used to build the Python APIs manually:
-```bash
-cd python/raft-dask
-python setup.py build_ext --inplace
-python setup.py install
-
-cd python/pylibraft
-python setup.py build_ext --inplace
-python setup.py install
+# to build pylibraft
+./build.sh libraft pylibraft --install --compile-libs
```

Comment:
I think it's great to have the build.sh variants in here as well but i'd like to also keep the manual build instructions for developers to know that the other method can be used as well. I think it would be best to include both, and maybe move the setup.py can also be used to build the Python APIs manuall after the build.sh instructions?